core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -74,8 +74,7 @@ case class LogReadResult(info: FetchDataInfo,
   }
 
   override def toString =
-    s"Fetch Data: [$info], HW: [$hw], leaderLogEndOffset: [$leaderLogEndOffset], readSize: [$readSize], error: [$error]"
-
+    s"Fetch Data: [$info], HW: [$hw], leaderLogEndOffset: [$leaderLogEndOffset], readSize: [$readSize], error: [${error.exceptionName}]"
```

Comment:
We're not very consistent, I think we do both. Personally, I think the error makes more sense in cases like this. `error: InvalidPartitionsException` seems less nice than `error: INVALID_PARTITIONS`.